src/styles/globals.css

```diff
@@ -2,10 +2,14 @@
   --font-family: 'Inter', sans-serif; /* Font family */
   --primary-color: #000000; /* Primary color */
   --secondary-color: #6c757d; /* Secondary color */
-  --background-color: #f8f9fa; /* Background color */
+  --background-color: #EBEBEB; /* Background color */
   --text-color: #343a40; /* Text color */
   --border-radius: 0.25rem; /* Border radius */
   --box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Box shadow */
+  --border-color: rgba(128, 128, 128, 0.1);/*Border color*/
   --transition-duration: 0.3s; /* Transition duration */
-  --font-weight: 700;
+  --font-weight: 700;/* Set font weight to 700 */
+  --font-size: 40px;/* Set font size to 40px */
+  --line-height: 48px; /* Set line height to 48.41px */
```

Comment:
Could you clean up this file by removing the comments? I don't think they're necessary in this case since it's already clear what the values are from the variable names